README.md

```diff
@@ -72,7 +72,7 @@ And end up with an object like this:
 var fm = require('front-matter')
 ```
 
-## fm(string)
+## fm(string, allowUnsafe = false)
```

Comment:
Would you mind making the second param an object for options, e.g.

    fm(string, { allowUnsafe: true })

This does two things that I think are desirable
- Helps with readability, making it really clear at the call site what option is being called
- The options object will make it easier to add more options in the future